test/test_dataloader.py

```diff
@@ -2173,26 +2173,46 @@ def test_basics(self):
         self.assertEqual(list(dl), list(dl2))
         self.assertEqual(list(dl), list(dl2_threading))
 
+    class Sorter(IterDataPipe):
+        def __init__(self, datapipe):
+            self.datapipe = datapipe
+
+        def __iter__(self):
+            return iter(sorted(self.datapipe))
+
     def test_shuffle(self):
         items = list(range(1000))
         dp = IterableWrapper(items).sharding_filter().shuffle()
 
         dl = DataLoader2(dp, batch_size=None, num_workers=2, shuffle=False)
         self.assertEqual(items, list(dl))
 
-        dl = DataLoader(dp, batch_size=None, num_workers=2, shuffle=False,
+        dl = DataLoader2(dp, batch_size=None, num_workers=2, shuffle=False,
```

Comment:
I'm guessing this was a mistake given that this test case is specifically for `DataLoader2`?